src/langs/assemblyscript/build.js

```diff
@@ -28,5 +26,7 @@ const buildLoader = async (buildDir) => {
   await copyFile(`${__dirname}/wasm-loader.js`, `${buildDir}/wheel-part-assemblyscript.wasm-loader.js`);
 };
 
-exports.buildLoader = buildLoader;
-exports.buildWasm = buildWasm;
\ No newline at end of file
+exports = {
```

Comment:
Which Node.js version do you use? When building with Docker (using Node.js 8), I get `buildLoader is not a function`. If I change this line to `module.exports`, it works fine.

Reply:
I use node 11. But this should works with 10 also